### PR TITLE
User guide for Ansys Python Manager

### PR DESCRIPTION
**Pull request by abhishekchitwar**
@RobPasMue 

Added an article as markdown file under docs for now.

### CODE REVIEW

**Comment by RobPasMue** [comment]
@abhishekchitwar - transform it to RST and place it in the landing page of our docs. It won't be an article itself. It will be our main landing page for the docs. Or "User guide", as you prefer.

**Comment by abhishekchitwar** [comment]
@RobPasMue 

Added as User Guide and moved relevant image data to _static under docs.

File: doc/source/user_guide.rst [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+development, and workflow automation. PyAnsys is a set of Python package
+that aligns with this trend, offering a Pythonic interface to Ansys
+Products. It serves as a powerful tool for automating engineering
+workflows, conducting parametric studies, and post-processing simulation
+results, among other applications. However, configuring Python
+environments and handling dependencies can occasionally prove to be
+daunting. This is where Ansys Python Manager comes to play.
+
+What is ANSYS python manager?
+=============================
```
Comment (RobPasMue) [inline doc/source/user_guide.rst:18]:
```suggestion
What is the Ansys Python Manager?
=================================
```

File: doc/source/user_guide.rst [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+==================
+
+The Ansys Python Manager offers a user-friendly interface for Python
+installation. You can easily select the desired Python version from the
+provided drop-down menu. The manager provides two options: standard
+Python and conda-forge python. The standard Python option is the
+official Python distribution, while the conda-forge Python option uses the
+community-driven distribution used with products like Anaconda.
+
+
```
Comment (RobPasMue) [inline doc/source/user_guide.rst:57]:
```suggestion
```

File: doc/source/user_guide.rst [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+.. image:: _static/ansys_python_manager.PNG
+   :align: center
+   :height: 650
+   :width: 560
+   :alt: Screenshot of Ansys Python Manager
+
+
+Create virtual environments tab
+===============================
+The “Create Virtual Environments” tab in the Ansys Python Manager
```
Comment (RobPasMue) [inline doc/source/user_guide.rst:76]:
```suggestion

The “Create Virtual Environments” tab in the Ansys Python Manager
```

**Review by RobPasMue — APPROVED** [review]
Some minor comments! LGTM @abhishekchitwar - thanks for the contribution!

File: doc/source/user_guide.rst (hunk 1 above)
Comment (abhishekchitwar) [reply doc/source/user_guide.rst:18]: @RobPasMue 
vale config is not allowing it. Is there any way around for it?

Comment (RobPasMue) [reply doc/source/user_guide.rst:18]: Really? Let me check that by merging

**Pull request merged**